### ISSUE TITLE
Send to Local Folder takes a long time to finish

### ISSUE THREAD

**Issue by khwerz** [gpodder/gpodder#653]
I tend to copy files locally to another folder, which is just an old cellphone I use for podcasts, however this phone is rather old and writes slowly so if you send/copy many files, it will take a long time to finish, so this is not a problem with gpodder per se, but something that could be ameliorated with a progress tab for copying files to a directory. this could be run in the background while you keep syncing everything manually.

**Comment by elelay** [comment]
Yes, definitly. There has been one attempt to do it in #300. I'll try again at some point.
